src/assets/fonts.tsx

```diff
@@ -8,4 +8,26 @@ injectGlobal`
         font-weight: normal;
         font-style: normal;
     }
-`
\ No newline at end of file
+`
+
+const primaryFontFamily = `'ITC Avant Garde Gothic W04', 'AvantGardeGothicITCW01D 731075', 'AvantGardeGothicITCW01Dm', 'Helvetica', 'sans-serif'`,
+
+export const primary = {
+    fontFamily: primaryFontFamily,
+    style: `
+        font-family: ${primaryFontFamily};
+        -webkit-font-smoothing: antialiased;
```

Comment:
<img width="544" alt="screen shot 2017-02-24 at 5 27 41 pm" src="https://cloud.githubusercontent.com/assets/296775/23323447/97943e1a-fab6-11e6-8904-aba664704bc2.png">

so it's working for the `transition` css property but not `font-smoothing` which is leading me to think, `font-smoothing` is not included in the list of properties that need to be prefixed. I'll double-check how `styled-components` does it